compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -3416,8 +3408,10 @@ public void visit(BLangCompoundAssignment compoundAssignment) {
         // Create the index access expression. ex: c[$temp3$][$temp2$][$temp1$]
         BLangExpression var = varRef;
```

Comment:
Better not use 'var' as the variable name. But it's the already existing code though